test/e2e/tests/synchronous-injection/synchronous-injection.spec.ts

```diff
@@ -17,18 +18,17 @@ describe('The provider', function () {
     await withFixtures(
       {
         fixtures: new FixtureBuilder().build(),
-        title: this.test.title,
+        title: this.test?.fullTitle(),
       },
       async ({ driver }) => {
-        await unlockWallet(driver);
+        await loginWithBalanceValidation(driver);
         await openDapp(driver);
 
         const isExpectedChainIdTextRendered =
           await driver.isElementPresentAndVisible({
             tag: 'div',
             text: 'Chain Id: 0x539',
           });
```

Comment:
keep the selector in this spec temporarily, as as this is a minimum dapp that seems we don't use in other tests, and with very limited functionalities. Will migrate to POM if needed in the future